tests/sentry/receivers/test_onboarding.py

```diff
@@ -482,3 +484,130 @@ def test_onboarding_complete(self):
             ).count()
             == 1
         )
+
+    @patch("sentry.signals.first_event_with_minified_stack_trace_received.send_robust")
+    def test_first_event_without_minified_stack_trace_received(self, send_robust):
```

Comment:
@priscilawebdev Can you add tests for the functionality within `record_event_with_first_minified_stack_trace_for_project` since we are stubbing it out with this test?

Reply:
what do you mean? I'd rather have tests that test two different things separately